sdk/communication/communication-phone-numbers/samples-dev/siprouting/manageSipRoutingConfiguration.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { SipRoutingClient } from "@azure/communication-phone-numbers";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const buildClient = () => {
+  const connectionString =
+    process.env.COMMUNICATION_SAMPLES_CONNECTION_STRING ||
+    "endpoint=https://resourceName.communication.azure.net/;accessKey=test-key";
+  return new SipRoutingClient(connectionString);
+};
+
+export async function main() {
+  console.log("\n== SIP Routing Client Example ==\n");
+
+  // Build client
+  const client = buildClient();
+
+  // Clear
+  await client.setRoutes([]);
+  await client.setTrunks([]);
```

Comment:
Is this a common pattern we expect customers to do often? If so it feels like we should add a convenience method for clearing

Reply:
If not, we should remove these 2 lines

Reply:
Replaced by a comment.
`
// NOTE: Before running the example please make sure that the trunks and the routes are empty for specified connection string.
// Otherwise the example may fail on data constraints.

const connectionString = process.env.COMMUNICATION_SAMPLES_CONNECTION_STRING || "endpoint=https://resourceName.communication.azure.net/;accessKey=test-key";
`

Reply:
Also specific FQDNs in examples were replaced by placeholders like '<first trunk fqdn>'.